.github/boring-cyborg.yml

```diff
@@ -1,13 +1,33 @@
 ##### Labeler ##########################################################################################################
 # Enable "labeler" for your PR that would add labels to PRs based on the paths that are modified in the PR.
+# * matches anything but /
+# ** matches anything including /
+
 labelPRBasedOnFilePath:
-  # Add 'Type: Map' to any changes within 'maps' folder or any subfolders
+  # Add 'Type: Map' to any changes to .dmm files
   "Type: Map":
-    - maps/**/*
+    - '**.dmm'
 
-  # Add 'Type: Map' to any changes within 'maps' folder or any subfolders
+  # Add 'Type: Map' to any changes to image files
   "Type: Icon":
-    - icons/**/*
+    - '**.dmi'
+    - '**.png'
+    - '**.jpg'
+
+  # Add 'Type: Sound' to any change to sound files
+  "Type: Sound":
+    - '**.ogg'
+    - '**.mp3'
+    - '**.wav'
+
+  # Add 'Type: Fluff' to any change to *custom_items*
+  "Type: Fluff":
+    - '**custom_items**'
```

Comment:
This should check for VOREStation\code\modules\client\preference_setup\loadout\loadout_fluffitems_vr.dm